### PR TITLE
Chrstian Base Features

### PR DESCRIPTION
**Pull request by c-amstutz**
Yeah so this should've been done a week ago

### CODE REVIEW

File: eventPlannerApp/modules/events/editEventPage.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    form.location.data = event['location']
+    form.ownerUsername.data = event['ownerUsername']
+    form.accessType.data = event['accessType']
+    form.associatedSchool.data = event['associatedSchool']
+    form.creatorUsername.data = event['creatorUsername']
+
+    return render_template('events/editEvent.html', form=form, data=data)
+
+
+class EditEventForm(FlaskForm):
```
Comment (cmdevoto) [inline eventPlannerApp/modules/events/editEventPage.py:109]: this is intriguing me

**Review by cmdevoto — APPROVED** [review]
looks great!

File: eventPlannerApp/modules/events/editEventPage.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+from wtforms.validators import DataRequired, NumberRange
+
+import calendar
+from datetime import datetime
+
+from . import bp
+from ... import dbInterface
+
+
+@bp.route('/event/edit/<int:eventId>', methods=['GET', 'POST'])
```
Comment (cmdevoto) [inline eventPlannerApp/modules/events/editEventPage.py:15]: this just needs login required

Comment (cmdevoto) [reply eventPlannerApp/modules/events/editEventPage.py:15]: on that same thought, do we want to consider if everyone can edit or view event pages? should we have a way to stop people from accessing the url of private events?

Comment (cmdevoto) [reply eventPlannerApp/modules/events/editEventPage.py:15]: also should it be that anyone invited to an event can edit?

**Review by cmdevoto — CHANGES_REQUESTED** [review]

**Review by bdivney — APPROVED** [review]

**Pull request merged**